checkov/terraform/checks/terraform/terraform/__init__.py

```diff
@@ -0,0 +1,5 @@
+from os.path import dirname, basename, isfile, join
```

Comment:
I assume this is copied from the rest of the directories, but tbh I really hate this behavior - it inexplicitly imports all files from this directory.
Instead - can you just import the specific file you added `StateLock.py`?

(`from .StateLock import <check-class>` or `import StateLock`)